GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCCTNanopbHelpers.m

```diff
@@ -49,6 +50,37 @@
   return pbBytesArray;
 }
 
+#pragma mark - General purpose device info collectors
+
+#if TARGET_OS_IOS || TARGET_OS_TV
```

Comment:
We've abstracted all platforms ifdefs to GDTCORPlatform (except for a couple in GDTCORReachability). Because we must support so many platforms, please abstract this into a function in GDTCORPlatform that returns sane values regardless of the platform (iOS, iPadOS, macOS, watchOS, tvOS, iOS extensions).

Reply:
Makes sense. Have moved this change to GDTCORPlatform to make it more centralized.